core/renderers/common/info.ts

```diff
@@ -534,17 +534,17 @@ export class RenderInfo {
     }
 
     // Decide where the extra padding goes.
-    if (row.align === Align.LEFT) {
+    if (row.align === Align.LEFT && lastSpacer) {
       // Add padding to the end of the row.
       lastSpacer.width += missingSpace;
-    } else if (row.align === Align.CENTRE) {
+    } else if (row.align === Align.CENTRE && firstSpacer && lastSpacer) {
       // Split the padding between the beginning and end of the row.
       firstSpacer.width += missingSpace / 2;
       lastSpacer.width += missingSpace / 2;
-    } else if (row.align === Align.RIGHT) {
+    } else if (row.align === Align.RIGHT && firstSpacer) {
       // Add padding at the beginning of the row.
       firstSpacer.width += missingSpace;
-    } else {
+    } else if (lastSpacer) {
       // Default to left-aligning.
       lastSpacer.width += missingSpace;
     }
```

Comment:
Add a final `else` to catch the case where none of the above were true and it couldn't be left-aligned. Currently this can add `missingSpace` to `row.width` without ever adding it to one of the spacers.

Reply:
Added an `else { return; }`